src/Limenius/Liform/Serializer/Normalizer/FormErrorNormalizer.php

```diff
@@ -51,11 +51,19 @@ public function normalize($object, $format = null, array $context = [])
     /**
      * {@inheritdoc}
      */
-    public function supportsNormalization($data, $format = null)
+    public function supportsNormalization(mixed $data, ?string $format = null, array $context = []): bool
     {
         return $data instanceof FormInterface && $data->isSubmitted() && !$data->isValid();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getSupportedTypes(?string $format): array
+    {
+        return ['object' => true];
```

Comment:
I think it should be more precise. 
```suggestion
        return [Form:class];
```